pkg/infra/teams/team3/election_strat.go

```diff
@@ -1,21 +1,25 @@
 package team3
 
 import (
+	//"fmt"
+
 	"infra/game/agent"
+	"infra/game/commons"
+	"math"
+	"strconv"
+
 	// "infra/game/commons"
 	"infra/game/decision"
 	"infra/game/state"
 	"infra/logging"
 	"math/rand"
-	"sort"
-	"strconv"
 )
 
-var (
-	w1, w2      float64
-	pastHP      []int
-	pastStamina []int
-)
+var w1 = make(map[commons.ID]float64)
+var w2 = make(map[commons.ID]float64)
+
+var pastHP = make(map[commons.ID]int)
+var pastStamina = make(map[commons.ID]int)
```

Comment:
Minor change, but convention is this:

`var` is used before initialisation - when the initial value is unknown (such as with `var x float64`)
we use the 'tusk' operator (`:=`) for assigning when initialisation is known

Worth using `w1 := make(map...)`

Reply:
Also, there is a clash in variable name:

w1 is used here as a map and also in `calcW1` as a `float64` - this makes readability less clear

Reply:
yes good spot!